agentendpoint/config_task.go

```diff
@@ -201,7 +209,7 @@ func enforceConfigResourceState(ctx context.Context, res *resource, rCompliance
 	clog.Debugf(ctx, "Running step 'enforce state' on resource %q.", configResource.GetId())
 	// Only enforce resources that need it.
 	if res.InDesiredState() {
-		clog.Debugf(ctx, "No enforcement required for %q.", configResource.GetId())
+		clog.Debugf(ctx, "Enforce state: No enforcement required for %q.", configResource.GetId())
```

Comment:
Info?

Reply:
I purposely did not make this info as the check state output should make it clear, and this would log every "normal" run which i want to avoid.